stable/elastalert/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 description: ElastAlert is a simple framework for alerting on anomalies, spikes, or other patterns of interest from data in Elasticsearch.
 name: elastalert
-version: 0.5.0
+version: 0.5.1
```

Comment:
I suggest changing version to 0.7.0 since this is introducing new parameters, and also since 0.6.0 is going to be merged soon from another approved PR.